firebase-firestore/firebase-firestore.gradle

```diff
@@ -15,6 +15,9 @@
 plugins {
     id 'firebase-library'
     id 'com.google.protobuf'
+    id 'kotlin-android'
+    // need serialization plugin at least 1.6.20
```

Comment:
During my test, I found that I need to have `$kotlinVersion` to be 1.6.20 , so that `kotlinx-serialization-core` lib can provide me the `@Serializable` annotations, etc.  But the serialization plugin version can be quite low (even 1.3.72 works).

Since I need Kotlin to be 1.6.20 anyways, I have changed this plugin version to be `$kotlinVersion`.